runtime/near-vm-runner/src/runner.rs

```diff
@@ -27,6 +27,17 @@ fn check_method(module: &Module, method_name: &str) -> Result<(), VMError> {
     }
 }
 
+/// `run` does the following:
+/// - deserialize and validate the `code` binary (see `prepare::prepare_contract`)
+/// - inject gas counting into
```

Comment:
Unfinished sentence. We should also watch out for grammar, here and above it makes sense to say `deserializes` and `injects`. Below, it should be `instantiates`, `calls`, etc.

Reply:
I did so initially, but the sentence starts with: "`run` does" ... You must be right, I'll change it